build.sbt

```diff
@@ -0,0 +1,62 @@
+import BuildHelper._
+
+inThisBuild(
+  List(
+    organization := "dev.zio",
+    homepage := Some(url("https://zio.github.io/zio-s3/")),
+    licenses := List("Apache-2.0" -> url("http://www.apache.org/licenses/LICENSE-2.0")),
+    developers := List(
+      Developer("regis-leray", "Regis Leray", "regis.leray@gmail.com", url("https://github.com/regis-leray"))
+    ),
+    Test / fork := true,
+    parallelExecution in Test := false,
+    publishMavenStyle := true,
+    pgpPassphrase := sys.env.get("PGP_PASSWORD").map(_.toArray),
+    pgpPublicRing := file("/tmp/public.asc"),
+    pgpSecretRing := file("/tmp/secret.asc"),
+    scmInfo := Some(
+      ScmInfo(url("https://github.com/zio/zio-s3/"), "scm:git:git@github.com:zio/zio-s3.git")
+    )
+  )
+)
+
+ThisBuild / publishTo := sonatypePublishToBundle.value
+
+addCommandAlias("fmt", "all scalafmtSbt scalafmt test:scalafmt")
+addCommandAlias("check", "all scalafmtSbtCheck scalafmtCheck test:scalafmtCheck")
+
+lazy val `zio-s3` = project
+  .in(file("."))
+  .settings(
+    skip in publish := true,
+    libraryDependencies ++= Seq(
+      "dev.zio"                %% "zio"                         % "1.0.0-RC17",
+      "dev.zio"                %% "zio-streams"                 % "1.0.0-RC17",
+      "dev.zio"                %% "zio-nio"                     % "0.4.0",
+      "dev.zio"                %% "zio-interop-java"            % "1.1.0.0-RC6",
+      "dev.zio"                %% "zio-interop-reactivestreams" % "1.0.3.5-RC2",
+      "software.amazon.awssdk" % "s3"                           % "2.10.50",
```

Comment:
If we ever add scala-steward to this, the AWS dependency is going to be a giant pain :D May want to include mergify at that point and auto-merge scala-steward PRs :D

Reply:
I would be great if we had a offcial zio http client. I could replace this dependency

Reply:
@regis-leray Can you go with sttp until the client is ready?

@NeQuissimus I think we need mergify throughout the whole organization to take care of steward stuff :).